connectors/v2/nrkradio.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+/* global Connector */
+
+Connector.getArtistTrack = function () {
+	var text = $('.active.music .fnn-title').text();
+	var separator = this.findSeparator(text);
+
+	var artist = null;
+	var track = null;
+
+	if (separator !== null) {
+		artist = text.substr(0, separator.index).trim();
+		track = text.substr(separator.index + separator.length).trim();
+	}
+
+	return {artist: artist, track: track};
+};
+
+Connector.isPlaying = function () {
+	return true;
```

Comment:
There's no way to check if song is paused, so it's better to assume it's always playing.
Although, the `Connector.isPlaying` function implementation can be removed, because core supposes the song is playing by default.